x/ccv/provider/keeper/partial_set_security.go

```diff
@@ -343,24 +343,6 @@ func (k Keeper) ComputeNextValidators(ctx sdk.Context, chainID string, bondedVal
 		}
 	}
 
-	// take only the first `MaxValidatorRank` many validators; others are not allowed to validate
```

Comment:
Is the sorting above still needed?

Reply:
The sorting above is for the inactive vals, too, see the comment below
```
// if inactive validators are not allowed, only consider the first `MaxProviderConsensusValidators` validators
// since those are the ones that participate in consensus
```

In general, this is in principle called with a sorted list already, but sorting here seems reasonable as a defensive measure (since not having a sorted list here is pretty bad and breaks PoS)